integration_tests/steps/integration_tests.py

```diff
@@ -512,38 +512,24 @@ def mock_publish_json(*args):
     context.simulation.redis_connection._sub_callback_dict[channel](payload)
 
 
-@when('the simulation is able to transmit intermediate results')
+@when('the simulation is able to transmit intermediate and final results')
 def interm_results(context):
     context.interm_results_count = 0
 
-    def interm_res_count(_):
+    def interm_res_count(_, _1):
```

Comment:
Can you please explain the `_1` ?

Reply:
Yes, the signature of the `publish` method has changed for Kafka and requires one extra argument (the partition name, which is the simulation_id). Therefore the mock of the publish method should include 2 positional arguments, for which we do not care for either of them. Therefore the Pythonic way of setting arguments that you actually do not need is this: _, _1, _2, _3 ....